config/database.yml

```diff
@@ -8,6 +8,8 @@ defaults: &defaults
   pool: 5
   timeout: 5000
   host: localhost
+  variables:
+    statement_timeout: 250
```

Comment:
I have not -- and I think it might be better to remove it (and change the specs a bit since they rely on this), the reason being it does not match production, where I did not commit any statement timeout variables:

https://github.com/18F/identity-devops/pull/524/files